app/controllers/users.js

```diff
@@ -68,3 +69,14 @@ exports.adminUserRegistration = (req, res, next) => {
     )
     .catch(next);
 };
+
+exports.getUserAlbumsList = (req, res, next) => {
+  let userId = req.session.id;
+  if (req.session.role === 'admin') {
+    userId = req.params.id ? req.params.id : req.session.id;
```

Comment:
Here you can use the optional parameter `||`
`userId = req.params.id || req.session.id`
In case `req.params.id` is undefined, the optional value will be set.